src/ReusableRelayBuilder.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Relay;
+
+use ArrayObject;
+use InvalidArgumentException;
+
+class ReusableRelayBuilder
```

Comment:
The "reusable" indicates it's building a "reusable relay" (as vs a single-use relay). Note that Relay will become Runner in the next iteration, and ReusableRelay will become Relay.